getting-started/security_policy_examples.md

```diff
@@ -71,36 +71,38 @@ Here, we demonstrate how to define security policies using our example microserv
 
     * Explanation: As the extension of the previous policy, we want to block all executables in the '/usr' directory and its subdirectories \(e.g., '/usr/bin', '/usr/sbin', and '/usr/local/bin'\). Thus, we add 'recursive: true' to extend the scope of the policy.
 
-    * Verification: After applying this policy, please get into the container with the 'ubuntu-2' label and run '/usr/bin/env' or '/usr/bin/whoami'. You will see that those commands are blocked. 
+    * Verification: After applying this policy, please get into the container with the 'ubuntu-2' label and run '/usr/bin/env' or '/usr/bin/whoami'. You will see that those commands are blocked.
 
-  * Allow specific executables only \([ksp-ubuntu-3-proc-dir-allow.yaml](../examples/multiubuntu/security-policies/ksp-ubuntu-3-proc-dir-allow.yaml)\)
+  * Allow specific executables to access certain files only \([ksp-ubuntu-3-file-dir-allow-from-source-path.yaml](../examples/multiubuntu/security-policies/ksp-ubuntu-3-file-dir-allow-from-source-path.yaml)\)
 
     ```text
     apiVersion: security.kubearmor.com/v1
     kind: KubeArmorPolicy
     metadata:
-      name: ksp-ubuntu-3-proc-dir-allow
+      name: ksp-ubuntu-3-file-dir-allow-from-source-path
       namespace: multiubuntu
     spec:
+      severity: 10
+      message: "a critical directory was accessed"
+      tags:
+      - WARNNING
```

Comment:
```suggestion
      - WARNING
```

Reply:
fixed the typo in 'security_policy_examples.md' and other files too.